src/Command/CodeStudio/CodeStudioPipelinesMigrateCommand.php

```diff
@@ -0,0 +1,612 @@
+    // We may already be authenticated with Acquia Cloud Platform via a refresh token.
+    // But, we specifically need an API Token key-pair of Code Studio.
+    // So we reauthenticate to be sure we're using the provided credentials.
+    $this->reAuthenticate($cloud_key, $cloud_secret, $this->cloudCredentials->getBaseUri());
+
+    $this->checklist = new Checklist($output);
+    $this->appUuid = $this->determineCloudApplication();
+    // Get Cloud application.
+    $cloud_application = $this->getCloudApplication($this->appUuid);
+    $this->gitLabProjectDescription = "Source repository for Acquia Cloud Platform application <comment>$this->appUuid</comment>";
+    $project = $this->determineGitLabProject($cloud_application);
+
+    // Migrate acquia-pipeline file
+    $this->getGitLabCiCdVariables($project, $this->appUuid, $cloud_key, $cloud_secret);
+    $this->checkPipelineExists($project);
+    $this->migrateCSStandardTemplate();
+    $this->migrateVariables();
+    $this->migrateBuild();
+    $this->migratePostDeploy();
+    $this->io->success([
+      "",
+      "*********** Migration completed successfully. *********\n",
+      "Created .gitlab-ci.yml and removed acquia-pipeline.yml file.\n",
+      "In order to run Pipeline, push .gitlab-ci.yaml to Main branch of Code Studio project.\n",
+      "Check your pipeline is running in Code Studio for your project.\n"
+    ]);
+
+    // Get Cloud account.
+    $acquia_cloud_client = $this->cloudApiClientService->getClient();
+    $account_adapter = new Account($acquia_cloud_client);
+    $account = $account_adapter->get();
+    $this->validateRequiredCloudPermissions($acquia_cloud_client, $this->appUuid, $account);
+    return 0;
+  }
+
+  /**
+   * @param \Symfony\Component\Console\Input\InputInterface $input
+   *
+   * @return bool
+   */
+  protected function commandRequiresAuthentication(InputInterface $input): bool {
+    return FALSE;
+  }
+
+  /**
+   * @throws \Acquia\Cli\Exception\AcquiaCliException
+   */
+  protected function validateEnvironment() {
+    if (!getenv('GITLAB_HOST')) {
+      throw new AcquiaCliException('The GITLAB_HOST environmental variable must be set.');
+    }
+  }
+
+  /**
+   * Search for existing project in code studio.
+   * @param ApplicationResponse $cloud_application
+   *
+   * @return array
+   */
+  protected function determineGitLabProject(ApplicationResponse $cloud_application) {
+    $projects = $this->gitLabClient->projects()->all(['search' => $cloud_application->uuid]);
+    if (count($projects) == 1) {
+      return reset($projects);
+    }
+    else {
+      throw new AcquiaCliException("[ERROR] '{$cloud_application->name}' is not configured as Code Studio project.\n [Help] You can configure the '{$cloud_application->name}' project in Code Studio by using 'acli cs:wizard' command \n You can find Code Studio documentation at https://docs.acquia.com/code-studio/getting-started/#create-code-studio-project \n");
```

Comment:
We don't add formatting to exception messages.

Reply:
I wanted to add help along with error message then how should I add it?

Reply:
You'd need to catch the exception here: https://github.com/acquia/cli/blob/master/src/EventListener/ExceptionListener.php#L54